### DISCUSSION TITLE
[求助] 这里能悄咪咪地发内鬼消息吗（坏笑）

### DISCUSSION THREAD

**Discussion by qwq1949** [share121/inter-knot#995] [其他]
## 发了会不会被mhy大手抓上法庭暴扣50万qwq

**Comment by Guivov** [comment]
真的假的O.o
发一个看看实力(坏笑

**Reply by qwq1949** [reply]
先v我50验证真人再发 .-.

**Comment by capacitor1** [comment]
不清楚。要不楼主先试试？doge

**Comment by sterve521** [comment]
真的假的O.o
发一个看看实力(坏笑

**Comment by kallen1314** [comment]
不是有内鬼网么？感觉没必要在这里发，自己去内鬼网看就可以了

**Reply by Guivov** [reply]
内鬼网是什么Σ(っ °Д °;)っ求一个

**Comment by Guivov** [comment]
内鬼网是什么Σ(っ °Д °;)っ求一个

**Reply by qwq1949** [reply]
主要是 telegram 或 Reddit
guarleaks和fireflyleaks群
不挂__可以直接去贴吧找搬运的
###### 嘘 别说是我说的（）

**Comment by lanbaixiang** [comment]
赶紧发，发了我好去举报

**Reply by qwq1949** [reply]
补药遮阳啊
你怎么辣么坏[😭](https://www.emojiall.com/zh-hans/emoji/%F0%9F%98%AD)（）
